pkg/apis/go.mod

```diff
@@ -2,6 +2,9 @@ module github.com/rancher/rancher/pkg/apis
 
 go 1.19
 
+// wrangler bracnhes need to be updated before replace can be removed
+replace github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230629203936-0e36fee7aea5
```

Comment:
We are still looking at the desired workflow for tagging Wrangler, but our current model requires this line until Wrangler has been released.